src/injectscripts/jobs/view.js

```diff
@@ -6,37 +6,60 @@ console.log("Running content script");
 
 //if ops logs update
 masterViewModel.notesViewModel.opsLogEntries.subscribe(function(d) {
-  cleanupBr();
+  lighthouseKeeper();
 });
 
 //if messages update
 masterViewModel.messagesViewModel.messages.subscribe(function(d) {
-  cleanupBr();
+  lighthouseKeeper();
 });
 
 //call on run
-cleanupBr();
+lighthouseKeeper();
 
 
 document.title = "#"+jobId;
 
-function cleanupBr() {
-  console.log("BR cleanup called")
-  //only run if messages and notes have loaded in (causes problems overwise and won't load)
-  var selector = '.job-details-page div[data-bind="foreach: opsLogEntries"] div[data-bind="text: Text"]';
+function lighthouseKeeper(){
 
-  $(selector).each(function() {
-    var text = $(this).html();
-    var replaced = text.replace(/&lt;br&gt;/g, '<br />');
-    $(this).html(replaced);
+  var $targetElements = $('.job-details-page div[data-bind="foreach: opsLogEntries"] div[data-bind="text: Text"]');
+
+  var ICEMS_Dictionary = {
+    'ASNSW'   : 'NSW Ambulance' ,
+    'FRNSW'   : 'NSW Fire &amp; Rescue' ,
+    'NSWPF'   : 'NSW Police Force' ,
+
+    'AA'      : 'As Above' ,
+    'INFT'    : 'Informant/Caller' ,
+    '\dPOBS?' : 'Person(s) On Board' ,
+    'POIS?'   : 'Person(s) Of Interest' ,
+    'POSS'    : 'Possible' ,
+    'PTS?'    : 'Patient(s)' ,
+    'VEHS?'   : 'Vehicle(s)' ,
+    'VICT?'   : 'Victim' ,
+    'Y[OR]'   : 'Years Old'
+  };
+
+  $targetElements.each(function(v){
```

Comment:
This all seems safe after careful review. Keep in mind potential XSS for future changes to this though.